db/system_distributed_keyspace.cc

```diff
@@ -335,17 +337,33 @@ static set_type_impl::native_type prepare_cdc_streams(const std::vector<cdc::str
     return ret;
 }
 
+// cdc_streams_descriptions mutation
+static future<mutation> get_mutation(
+        const database& db,
+        db_clock::time_point time,
+        const std::vector<cdc::stream_id>& streams) {
+    auto s = db.find_schema(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_DESC);
+    mutation m(s, partition_key::from_singular(*s, time));
+    m.set_cell(clustering_key::make_empty(), to_bytes("streams"),
+            make_set_value(cdc_streams_set_type, prepare_cdc_streams(streams)), api::new_timestamp());
+    co_return m;
+}
+
 future<>
 system_distributed_keyspace::create_cdc_desc(
         db_clock::time_point time,
         const std::vector<cdc::stream_id>& streams,
         context ctx) {
-    return _qp.execute_internal(
-            format("INSERT INTO {}.{} (time, streams) VALUES (?,?)", NAME, CDC_DESC),
+    using namespace std::chrono_literals;
+    auto m = co_await get_mutation(_qp.db(), time, streams);
+    co_return co_await _sp.mutate(
+            { std::move(m) },
             quorum_if_many(ctx.num_token_owners),
-            internal_distributed_timeout_config,
-            { time, make_set_value(cdc_streams_set_type, prepare_cdc_streams(streams)) },
-            false).discard_result();
+            db::timeout_clock::now() + 10s,
```

Comment:
why +10s?

Reply:
Everything else in `system_distributed_keyspace` uses `10s`

Reply:
Ok. What's the meaning of it? Timeout?

Reply:
Yes. Previously it was expressed using `internal_distributed_timeout_config`